{"name": "Accessing Product Page", "status": "broken", "statusDetails": {"message": "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=59955): Max retries exceeded with url: /session/713b9d3234605b378ca9dd6cd7010330/url (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001AF94048E60>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))\n", "trace": "  File \"C:\\Users\\ttravis\\OneDrive - QA Consultants\\Desktop\\TrainingProject\\.env\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\ttravis\\OneDrive - QA Consultants\\Desktop\\TrainingProject\\.env\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\steps.py\", line 62, in step_impl\n    context.driver.get(\"https://www.automationexercise.com/products\")\n  File \"C:\\Users\\ttravis\\OneDrive - QA Consultants\\Desktop\\TrainingProject\\.env\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 356, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"C:\\Users\\ttravis\\OneDrive - QA Consultants\\Desktop\\TrainingProject\\.env\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 345, in execute\n    response = self.command_executor.execute(driver_command, params)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ttravis\\OneDrive - QA Consultants\\Desktop\\TrainingProject\\.env\\Lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 302, in execute\n    return self._request(command_info[0], url, body=data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ttravis\\OneDrive - QA Consultants\\Desktop\\TrainingProject\\.env\\Lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 322, in _request\n    response = self._conn.request(method, url, body=body, headers=headers)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ttravis\\OneDrive - QA Consultants\\Desktop\\TrainingProject\\.env\\Lib\\site-packages\\urllib3\\_request_methods.py\", line 144, in request\n    return self.request_encode_body(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ttravis\\OneDrive - QA Consultants\\Desktop\\TrainingProject\\.env\\Lib\\site-packages\\urllib3\\_request_methods.py\", line 279, in request_encode_body\n    return self.urlopen(method, url, **extra_kw)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ttravis\\OneDrive - QA Consultants\\Desktop\\TrainingProject\\.env\\Lib\\site-packages\\urllib3\\poolmanager.py\", line 444, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ttravis\\OneDrive - QA Consultants\\Desktop\\TrainingProject\\.env\\Lib\\site-packages\\urllib3\\connectionpool.py\", line 877, in urlopen\n    return self.urlopen(\n           ^^^^^^^^^^^^^\n  File \"C:\\Users\\ttravis\\OneDrive - QA Consultants\\Desktop\\TrainingProject\\.env\\Lib\\site-packages\\urllib3\\connectionpool.py\", line 877, in urlopen\n    return self.urlopen(\n           ^^^^^^^^^^^^^\n  File \"C:\\Users\\ttravis\\OneDrive - QA Consultants\\Desktop\\TrainingProject\\.env\\Lib\\site-packages\\urllib3\\connectionpool.py\", line 877, in urlopen\n    return self.urlopen(\n           ^^^^^^^^^^^^^\n  File \"C:\\Users\\ttravis\\OneDrive - QA Consultants\\Desktop\\TrainingProject\\.env\\Lib\\site-packages\\urllib3\\connectionpool.py\", line 847, in urlopen\n    retries = retries.increment(\n              ^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ttravis\\OneDrive - QA Consultants\\Desktop\\TrainingProject\\.env\\Lib\\site-packages\\urllib3\\util\\retry.py\", line 515, in increment\n    raise MaxRetryError(_pool, url, reason) from reason  # type: ignore[arg-type]\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "steps": [{"name": "Given I am on the login page", "status": "passed", "start": 1716555522098, "stop": 1716555526223}, {"name": "When I log in with valid credentials", "status": "passed", "start": 1716555526223, "stop": 1716555531377}, {"name": "Then I should see the delete account link", "status": "passed", "start": 1716555531377, "stop": 1716555534210}, {"name": "Then I will go to the products page", "status": "broken", "statusDetails": {"message": "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=59955): Max retries exceeded with url: /session/713b9d3234605b378ca9dd6cd7010330/url (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001AF94048E60>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))\n", "trace": "  File \"C:\\Users\\ttravis\\OneDrive - QA Consultants\\Desktop\\TrainingProject\\.env\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\ttravis\\OneDrive - QA Consultants\\Desktop\\TrainingProject\\.env\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\steps.py\", line 62, in step_impl\n    context.driver.get(\"https://www.automationexercise.com/products\")\n  File \"C:\\Users\\ttravis\\OneDrive - QA Consultants\\Desktop\\TrainingProject\\.env\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 356, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"C:\\Users\\ttravis\\OneDrive - QA Consultants\\Desktop\\TrainingProject\\.env\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 345, in execute\n    response = self.command_executor.execute(driver_command, params)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ttravis\\OneDrive - QA Consultants\\Desktop\\TrainingProject\\.env\\Lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 302, in execute\n    return self._request(command_info[0], url, body=data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ttravis\\OneDrive - QA Consultants\\Desktop\\TrainingProject\\.env\\Lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 322, in _request\n    response = self._conn.request(method, url, body=body, headers=headers)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ttravis\\OneDrive - QA Consultants\\Desktop\\TrainingProject\\.env\\Lib\\site-packages\\urllib3\\_request_methods.py\", line 144, in request\n    return self.request_encode_body(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ttravis\\OneDrive - QA Consultants\\Desktop\\TrainingProject\\.env\\Lib\\site-packages\\urllib3\\_request_methods.py\", line 279, in request_encode_body\n    return self.urlopen(method, url, **extra_kw)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ttravis\\OneDrive - QA Consultants\\Desktop\\TrainingProject\\.env\\Lib\\site-packages\\urllib3\\poolmanager.py\", line 444, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ttravis\\OneDrive - QA Consultants\\Desktop\\TrainingProject\\.env\\Lib\\site-packages\\urllib3\\connectionpool.py\", line 877, in urlopen\n    return self.urlopen(\n           ^^^^^^^^^^^^^\n  File \"C:\\Users\\ttravis\\OneDrive - QA Consultants\\Desktop\\TrainingProject\\.env\\Lib\\site-packages\\urllib3\\connectionpool.py\", line 877, in urlopen\n    return self.urlopen(\n           ^^^^^^^^^^^^^\n  File \"C:\\Users\\ttravis\\OneDrive - QA Consultants\\Desktop\\TrainingProject\\.env\\Lib\\site-packages\\urllib3\\connectionpool.py\", line 877, in urlopen\n    return self.urlopen(\n           ^^^^^^^^^^^^^\n  File \"C:\\Users\\ttravis\\OneDrive - QA Consultants\\Desktop\\TrainingProject\\.env\\Lib\\site-packages\\urllib3\\connectionpool.py\", line 847, in urlopen\n    retries = retries.increment(\n              ^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ttravis\\OneDrive - QA Consultants\\Desktop\\TrainingProject\\.env\\Lib\\site-packages\\urllib3\\util\\retry.py\", line 515, in increment\n    raise MaxRetryError(_pool, url, reason) from reason  # type: ignore[arg-type]\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1716555534210, "stop": 1716555550559}], "start": 1716555522098, "stop": 1716555550585, "uuid": "26ac0ec3-a9ac-41c0-bc45-1c80176a2169", "historyId": "bc32ae7114fdea4736668e464864bef3", "fullName": "Online Shopping Testing: Accessing Product Page", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "productsTest"}, {"name": "feature", "value": "Online Shopping Testing"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}